lib/gold.ex

```diff
@@ -12,18 +12,18 @@ defmodule Gold do
   @doc """
   Starts GenServer link with Gold server.
   """
-  def start_link(config), do: GenServer.start_link(__MODULE__, config)
+  def start_link(config, opts \\ []), do: GenServer.start_link(__MODULE__, config, opts)
```

Comment:
I'm all :+1: for this as well. Being able to pass options is a standard, eg, for `name: :ing` the process...